src/Analyzer/Resolve/QueryAnalyzer.cpp

```diff
@@ -4505,7 +4508,37 @@ void QueryAnalyzer::resolveTableFunction(QueryTreeNodePtr & table_function_node,
             table_name = table_identifier[1];
         }
 
-        auto parametrized_view_storage = scope_context->getQueryContext()->buildParametrizedViewStorage(function_ast, database_name, table_name);
+        auto context = scope_context->getQueryContext();
+        auto param_values_result = analyzeFunctionParamValues(function_ast, context, &scope.aliases);
+
+        for (const auto & [param_name, alias] : param_values_result.unresolved_param_aliases)
+        {
+            auto it = scope.aliases.alias_name_to_expression_node_before_group_by.find(alias);
```

Comment:
I'd say I do need to do this, because I need to be able to update `param_values_result.resolved_param_values` with newly resolved values exactly at this point before calling `context->buildParametrizedViewStorage`, otherwise it will fail with "parameter not set".